Process.h

```diff
@@ -49,6 +49,7 @@ typedef enum ProcessFields {
    NLWP = 51,
    TGID = 52,
    PERCENT_NORM_CPU = 53,
+   TRANSLATED = 54,
```

Comment:
This should probably not go into the cross-platform `Process` struct, but in the darwin specific `DarwinProcess` one

Reply:
Since DarwinProcess was not yet made ready to writeField/compare based on DarwinProcess-specific data, this required a bit more extensive changes. Pushed.